### PR TITLE
AccountPickScreen: Ask for confirmation when pressing on 'remove account'

### PR DESCRIPTION
**Pull request by AlaaElden98**
Fixes part of #4651

### CODE REVIEW

File: src/account/AccountPickScreen.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /* @flow strict-local */
 
 import React, { PureComponent } from 'react';
-
```
Comment (chrisbobbe) [inline src/account/AccountPickScreen.js:4]: nit: Keep a blank line, and keep third-party imports (the one from 'react' and the new one from 'react-native') above the blank line.

File: src/account/AccountPickScreen.js [hunk 2]
```diff
@@ -36,7 +36,20 @@ class AccountPickScreen extends PureComponent<Props> {
   };
 
   handleAccountRemove = (index: number) => {
-    this.props.dispatch(removeAccount(index));
+    Alert.alert(
+      'Remove Account?',
```
Comment (chrisbobbe) [inline src/account/AccountPickScreen.js:40]: Let's be sure to wire these new user-facing strings up for translation, by using `_` (see the jsdoc on our `GetText` type for how to get access to that in a class component, and examples like `UserStatusScreen` or `LanguagePicker`) and adding the new strings to messages_en.json.

Also, let's go for the same capitalization, punctuation, and button style as we use for the logout confirmation in src/account-info/ProfileScreen.js.

**Review by chrisbobbe — CHANGES_REQUESTED** [review]
Thanks, @AlaaElden98! I'll be glad to see a confirmation here; it seems disruptive for the device to totally forget about an account without checking to make sure that's actually what the user wanted to do.

I have a commit message nit:

```
AccountPickScreen: Ask for confirmation

Ask for confirmation when the user press on the remove button

Fixes part of: #4651
```

The summary line has room to be more specific than that; what do you think about something like `AccountPickScreen: Confirm before removing account from device.`? Then the "Ask for confirmation..." line in the body can disappear because it's already been said in the summary.

**Comment by chrisbobbe** [comment]
Thanks! Merged, with a few tweaks; see the commits I added after yours, and also I put a note mentioning #4651 back into your commit. 🙂

**Comment by AlaaElden98** [comment]
> Thanks! Merged, with a few tweaks; see the commits I added after yours, and also I put a note mentioning #4651 back into your commit. 🙂

Noted, thanks!  
Is the rules of adding blank lines between codes exists in our docs 🤔

**Comment by gnprice** [comment]
> Is the rules of adding blank lines between codes exists in our docs

The basic rule here is to use blank lines to separate things that don't belong grouped together. There isn't a formula for what belongs grouped together, so it's hard to codify in a rule. In general a good practice is to look at other code (in the same file, or the same codebase) and follow what it's doing.

**Comment by gnprice** [comment]
One source that may be helpful to read through is the Google JavaScript style guide. We don't necessarily align with all the choices they make, but probably the large majority of what's there is good advice in our codebase or any other JS codebase. Here's what they say about blank lines:
https://google.github.io/styleguide/jsguide.html#formatting-vertical-whitespace
and it's pretty close to what we do, and in particular explains why we have one here.

**Pull request merged**